src/openklant/components/klantinteracties/api/viewsets/klantcontacten.py

```diff
@@ -554,7 +555,9 @@ def perform_destroy(self, instance):
         description="Maak een KlantContact, Betrokkene en een OnderwerpObject aan.",
     ),
 )
-class MaakKlantcontactViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
+class MaakKlantcontactViewSet(
+    CheckQueryParamsMixin, mixins.CreateModelMixin, viewsets.GenericViewSet
```

Comment:
This viewset is POST only and doesn't have any query params, so let's not include this
```suggestion
    mixins.CreateModelMixin, viewsets.GenericViewSet
```